server/mergin/auth/controller.py

```diff
@@ -423,11 +423,11 @@ def update_user(username):  # pylint: disable=W0613,W0612
 @auth_required(permissions=["admin"])
 def delete_user(username):  # pylint: disable=W0613,W0612
     user = User.query.filter_by(username=username).first_or_404("User not found")
-    user_deleted.send(user)
     inactivate_user(user)
-    db.session.delete(user)
-    db.session.commit()
-    return "", 200
+    user_account_closed.send(user)
```

Comment:
Do we need emit this event?

Reply:
we do, there is a listener which removes workspace related bits (like references in workspace tables)